Code/src/main/java/nl/utwente/group10/ui/components/DisplayBlock.java

```diff
@@ -81,11 +81,11 @@ public Expr asExpr() {
     }
 
     public void invalidate() {
-        try {
+    	try {
             GhciSession ghci = GhciSession.getInstance();
             setOutput(ghci.pull(inputAnchor.asExpr()));
         } catch (GhciException e) {
-            setOutput("???");
+        	setOutput("???");
         }
     }
 }
```

Comment:
Je hebt in deze file ook de whitespace aangepast, maar ik vind het origineel beter.